common/src/model_registration/register.py

```diff
@@ -92,19 +99,17 @@ def main(args):
     model_path = args.model_path
     model_info_path = args.model_info_path
     registration_details = args.registration_details
+    tags,properties = {},{}
```

Comment:
tags = properties = {}

Reply:
Since dictionary is mutable in python.This will make tags and properties to point to same reference pointer(same address). 
So we need to initiate both at different address.